src/Widgets/NavigationWidget/NavigationWidgetEditingConfig.ts

```diff
@@ -3,9 +3,20 @@ import { NavigationWidget } from './NavigationWidgetClass'
 import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(NavigationWidget, {
-  attributes: { logOutLabel: { title: 'Log-out menu item label' } },
+  attributes: {
+    logOutLabel: { title: 'Log-out menu item label' },
+    slimDesign: {
+      title: 'Slim design?',
+      description: 'Shows only the meta navigation, not the "main" navigation.',
+    },
+  },
   title: 'Navigation',
   thumbnail: Thumbnail,
-  properties: ['metaNavigationObjs', 'searchInputLabel', 'logOutLabel'],
+  properties: [
+    'metaNavigationObjs',
+    'slimDesign',
+    'searchInputLabel',
```

Comment:
* [ ] The search is currently hidden if we use the slim nav. If this is intentional, let's please grey out the label property conditionally.

Reply:
=> https://github.com/Scrivito/scrivito-portal-app/pull/480